e2e/test/scenarios/dashboard/dashboard-card-resizing.cy.spec.js

```diff
@@ -0,0 +1,270 @@
+import _ from "underscore";
+import {
+  editDashboard,
+  resizeDashboardCard,
+  restore,
+  saveDashboard,
+  visitDashboard,
+} from "e2e/support/helpers";
+import { SAMPLE_DB_ID } from "e2e/support/cypress_data";
+import { SAMPLE_DATABASE } from "e2e/support/cypress_sample_database";
+
+import { GRID_WIDTH } from "metabase/lib/dashboard_grid";
+
+const VISUALIZATION_SIZES = {
```

Comment:
I think the idea is that we keep our implementation and testing code as separate as possible, so if there's any regressions in the size of the card in real life, our tests will still catch them. Then, the only way we can make a real change to the size of the cards is to modify the test to pass, in case someone messes with the implementation of `getDefaultSize` or `getMinSize` in the future